src/Apps/Conversation/Components/DetailsHeader.tsx

```diff
@@ -21,15 +23,23 @@ interface DetailsHeaderProps extends DetailsProps {}
 
 export const DetailsHeader: FC<DetailsHeaderProps> = props => {
   const { showDetails, setShowDetails } = props
+  const isMobile =
+    __internal__useMatchMedia(themeProps.mediaQueries.xs) ||
+    __internal__useMatchMedia(themeProps.mediaQueries.sm)
+
+  if (isMobile) {
```

Comment:
that's kinda okayish but I think of another way to use the responsive props, for example:
```jsx
return (
   <StackableBorderBox display={["none", "inherit"]}
   ...
```
is it possible for this case?

Reply:
I know you wrote in the PR 
```
The reason seems to be that using different values on the attributes (i.e.: display={["none", "none", "flex", "flex"] wasn't applying the changes at the same widths as breakpoints.
```
But wanted to double check if you've tried here

Reply:
nvm gh display messed up when I was taking a look here, I see it was removed 🤦